pages/components/Input/input.css

```diff
@@ -0,0 +1,47 @@
+.form-label {
+  margin-bottom: 0.5rem;
+}
+
+.form-control {
+  display: block;
+  width: 100%;
+  padding: 0.375rem 0.75rem;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: var(--primary-color);
+  background-color: var(--white-global);
+  background-clip: padding-box;
+  border: 1px solid #bbb;
+  border-radius: 0.25rem;
+}
```

Comment:
user color variables everywhere you have used #bbb. And use 2.5px as instead of 0.25rem as value is too small.

Reply:
Ack, done